experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -83,13 +86,48 @@ export const MOBILE_APPLICATIONS = [
   NimbusExperimentApplicationEnum.FOCUS_IOS,
 ];
 
-interface SelectExperimentOption {
+interface SelectExperimentBranchOption {
+  id: number;
   name: string;
   slug: string;
   description: string;
   value: string;
+  branchSlug: string | null;
```

Comment:
Do we need all three of `id`, `slug`, and `branchSlug` here? I don't think we really need to show the `slug` to the user 🤔  wdyt?

Reply:
The id is used for submitting to the API, the slug gets shown in the drop down.

Reply:
But why do we need to show the slug in the dropdown? Is there a reason why a user would need to see that? It seems superfluous

Reply:
Oh becuase often people identify things by the slug rather than the name, since the names can be similar or ambiguous but the slugs are exact.

Reply:
Awesome, makes sense 👍